unix/x0vncserver/x0vncserver.cxx

```diff
@@ -39,27 +39,12 @@
 #include <X11/X.h>
 #include <X11/Xlib.h>
 #include <X11/Xutil.h>
-#include <X11/XKBlib.h>
-#ifdef HAVE_XTEST
-#include <X11/extensions/XTest.h>
-#endif
-#ifdef HAVE_XDAMAGE
-#include <X11/extensions/Xdamage.h>
-#endif
-#ifdef HAVE_XFIXES
-#include <X11/extensions/Xfixes.h>
-#endif
 
 #include <x0vncserver/Geometry.h>
 #include <x0vncserver/Image.h>
 #include <x0vncserver/XPixelBuffer.h>
 #include <x0vncserver/PollingScheduler.h>
```

Comment:
I suspect more of these can be removed?

Reply:
Yes. In the updated request, I've moved 3 more, which are present in XDesktop.h. There are probably others as well (strings.h etc), but that's not related to moving XDesktop to a separate file.